config/routes.rb

```diff
@@ -174,7 +174,7 @@
   end
 
   namespace :dor do
-    match 'republish/:pid', action: :republish, via: %i[get post]
+    match 'republish/:pid', action: :republish, as: 'republish', via: %i[get post]
```

Comment:
as noted above, `dor_path` is not very descriptive since this is actually the republish path